modules/gatsby-theme-ocular/src/templates/doc-page-markdown.jsx

```diff
@@ -34,7 +34,7 @@ const CustomLinkWrapper = relativeLinks => {
       return <A href={href} {...props} />;
     }
     const hrefWithoutLeadingSlash = href.startsWith('/') ? href.slice(1) : href;
-    return <A {...props} href={relativeLinks[hrefWithoutLeadingSlash]} />;
+    return <A {...props} href={relativeLinks[hrefWithoutLeadingSlash]} || href/>;
```

Comment:
The fix is fine, however from a maintainability perspective, stashing behavior that requires a PR to fix inside an inline expression without comments is pretty subtle and not very self-documenting.

How about a helper function 

```js
function resolveLink(href) {
   const hrefWithoutLeadingSlash = href.startsWith('/') ? href.slice(1) : href;
   // dont touch links if a relative link to another doc is not found
   return relativeLinks[hrefWithoutLeadingSlash]} || href;
}
```

even better:
-  start extracting that function to a helper file `parse-links.js` or similar.